### DISCUSSION TITLE
Paypal username

### DISCUSSION THREAD

**Discussion by ACAwebbuilder** [code4recovery/12-step-meeting-list#1066] [Q&A]
Good morning!

I am wanting to use the paypal feature in the meeting. It asks for the Paypal Username. I have tried both the e-mail address and @username from the account and both send me to a page that says

"Sorry. It looks like nothing was found at this location. Maybe try one of the links below or a search?"

Does anyone have any ideas of what that field is expecting to be entered and where to get that information from the Paypal account?

**Comment by joshreisner** [comment]
oops sorry about that @ACAwebbuilder - looks like we have some confusion there, and need to get aligned internally

in some places we expect a string username, eg `MyGroupName`, and in some cases we expect a full PayPal URL, eg `https://www.paypal.com/paypalme/MyGroupName`

we'll create a ticket to figure this out!!

**Reply by joshreisner** [reply]
hi! to circle back on this: we have some work to do on our end but the decision is to go with PayPal.me usernames, eg `GroupName`

https://www.paypal.com/us/smarthelp/article/what-is-paypalme-help432

once we finish the work on our side this will start working properly, but if you want to go ahead and enter the data in the `GroupName` format you should do that
